src/Java.Interop.Tools.JavaTypeSystem/JavaModels/JavaFieldModel.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Java.Interop.Tools.JavaTypeSystem.Models
+{
+	public class JavaFieldModel : JavaMemberModel
+	{
+		public string Type { get; }
+		public string TypeGeneric { get; }
+		public string? Value { get; }
+		public bool IsTransient { get; }
+		public bool IsVolatile { get; }
+		public bool IsNotNull { get; }
+
+		public JavaTypeReference? TypeReference { get; private set; }
```

Comment:
What *is* this?  In keeping my earlier suggestion to use names as used by System.Reflection, would this be akin to `FieldInfo.FieldType`?  Something else?

Reply:
Renamed to `TypeModel` for consistency with other `Model` classes: `TypeModel` is the resolved object model for the `string Type`.